### DISCUSSION TITLE
1st version of 86Box?

### DISCUSSION THREAD

**Discussion by jadasse** [86Box/86Box#2178] [General]
Does anyone know where I can find the **first** version of 86Box (PCem-X) and the roms for it? I ask because I am curious.

**Comment by jriwanek** [comment]
You'd have to build it yourself, but: https://github.com/OBattler/PCem-X

**Reply by jadasse** [reply]
What about the compatible roms?

**Reply by jriwanek** [reply]
You'd have to dig around for those, but honestly, you can probably just use the 86box roms repo, specifically the older release versions to reconstruct it.

**Reply by jadasse** [reply]
> You'd have to build it yourself, but: https://github.com/OBattler/PCem-X

and building it would be the same as how one would build 86Box based on the docs?

**Reply by jriwanek** [reply]
No idea, I strongly suspect a lot has changed, but I've never tried.
We really can't provide support for such ancient versions, especially not just for curiosity's sake either.

**Reply by jadasse** [reply]
> No idea, I strongly suspect a lot has changed, but I've never tried. We really can't provide support for such ancient versions, especially not just for curiosity's sake either.

ok
